amundsen_application/static/js/components/common/AvatarLabel/index.tsx

```diff
@@ -11,11 +11,9 @@ export interface AvatarLabelProps {
 
 const AvatarLabel: React.SFC<AvatarLabelProps> = ({ label, src }) => {
   return (
-    <div className='avatar-label-component'>
-      <div id='component-avatar' className='component-avatar'>
-        <Avatar name={label} src={src} size={24} round={true} />
-      </div>
-      <label id='component-label' className='component-label'>{label}</label>
+    <div className="avatar-label-component">
```

Comment:
Can we revisit some of the style changes? The `AvatarLabels` as they present now have text that is not as well vertically centered as they used to be.

Reply:
Fixed. This was one of the few times where `vertical-align:middle` actually worked!